app/scripts/actions/index.js

```diff
@@ -14,6 +15,10 @@ export const LOGIN = 'LOGIN';
 export const LOGIN_INFLIGHT = 'LOGIN_INFLIGHT';
 export const LOGIN_ERROR = 'LOGIN_ERROR';
 
+export const ADD_CMR = 'ADD_CMR';
+export const ADD_CMR_ENVIRONMENT = 'ADD_CMR_ENVIRONMENT';
```

Comment:
Are `ADD_CMR_ENVIRONMENT` and `ADD_CMR_PROVIDER` being used anymore? If not we should remove them and the reducers for them

Reply:
They are not.    I can nuke them.